docs/GroupDefs.hpp

```diff
@@ -257,7 +257,7 @@
  * To compare two floating-point numbers that may differ by round-off, use the
  * helper object `approx`. This is an instance of Catch's comparison class
  * `Approx` in which the relative tolerance for comparisons is set to roughly
- * \f$10^{-14}\f$ (i.e. `std::numeric_limits<double>::epsilon()*100`).
+ * \f$10^{-14}\f$ (i.e. <tt>std::numeric_limits<double>::epsilon()*100</tt>).
```

Comment:
`<tt>` is not supported in [HTML5](https://www.w3schools.com/tags/tag_tt.asp). I don't see why the single tick doesn't work, I've used this extensively.